### ISSUE TITLE
Can't seem to save anything in operations

### ISSUE THREAD

**Issue by TB12AirJordan** [mitre/caldera#3057]
I have installed the manual version of 5.0.0
When I go to Adversaries and try to build a new Adversary, when I add an ability the save button doesn't work. No matter what it leaves the operation blank. Please let me know what logs would be helpful.

**Comment by uruwhy** [comment]
If you open up your browser developer tools and go to the developer console, what error(s) pop up when using the save button?

**Comment by TB12AirJordan** [comment]
I get a 200 back which means it is good. It saves the adversary but doesn't save any of the abilities. It is so weird.

**Comment by TB12AirJordan** [comment]
No, but the weird thing is, if I restart Caldera the changes are there. Like it saves it properly but it won't load properly until a restart of Caldera. So apparently the save button is working, it just doesn't load the changes until a restart of the service.

**Comment by uruwhy** [comment]
If you exit out of the adversary/operation GUI module and reopen it, do the changes appear then? Or is restarting the entire server the only way for you to see the changes?

**Comment by TB12AirJordan** [comment]
Yes restarting the browser and going back seems to work fine. I don't apparently need to restart the server.

**Comment by uruwhy** [comment]
I just tried on a fresh local installation of 5.0.0. It looks like you don't get any "successfully saved" message when you save a new adversary profile, but if I go to the operations tab and create a new operation, my new adversary shows up in the adversary list (no page refresh needed)

**Comment by TB12AirJordan** [comment]
Yes I am noticing that also. But that can't be expected behavior right? I mean when we click save I would think it should save and staying on the adversary page should be enough.

**Comment by uruwhy** [comment]
@TB12AirJordan Apologies for the late follow-up, but there have been recent [updates to the magma plugin](https://github.com/mitre/magma/pull/82) which may address this issue you've been seeing

**Comment by deacon-mp** [comment]
Thank you for the report\! This appears to be an environment or configuration issue. For help with saving operations and troubleshooting, please see the [Caldera documentation](https://caldera.readthedocs.io/) or open a discussion. Closing as there isn't enough information to reproduce this as a core Caldera bug — feel free to reopen with more details about your environment, Caldera version, and the exact steps to reproduce the issue.
